csv2docx/cli.py

```diff
@@ -32,15 +51,16 @@ def convert(data, template, delimiter):
         # see if all fields are accounted for in the .csv header
         column_in_data = set(docx_mergefields) - set(csv_headers)
         if len(column_in_data) > 0:
-            print (f"{column_in_data} is in the word document, but not csv.")
+            print(f"{column_in_data} is in the word document, but not csv.")
             return
 
         print("All fields are present in your csv. Generating Word docs ...")
 
+        path = create_path_if_not_exists(path)
+
         for counter, row in enumerate(csvdict):
             # Must create a new MailMerge for each file
             docx = MailMerge(template)
-            single_document = {key : row[key] for key in docx_mergefields}
+            single_document = {key: row[key] for key in docx_mergefields}
             docx.merge_templates([single_document], separator='page_break')
-            # TODO: write to user-defined subfolder
-            docx.write(f"{counter}.docx")
+            docx.write(f"{path}/{counter}.docx")
```

Comment:
Just had a thought that this might cause issues on windows given we are using `/`. However, because we convert `path` to type `pathlib.Path` it means that once we reach this `docx.write` that it is now a pathlib.path and not a string. This is _very important_ because if you use the backslash operator (`/`) after a pathlib.path object then it will join them 👍

Reply:
So that means the "/" will work in this case right? Also, the counter would ideally be a filename based on the .csv - see also #16 - but that might be for another PR

Reply:
Yes, it would work in this case, but I feel that we should probably instead create a variable before adding it to `docx.write` to make it more explicit, e.g. `write(f"{path}.docx")` where path is defined before 👍